### DISCUSSION TITLE
Inaccurate information about required ports?

### DISCUSSION THREAD

**Discussion by camaeel** [siderolabs/talos#7544] [General]
When I tried to setup a new cluster I opened ports as described here: https://www.talos.dev/v1.4/learn-more/talos-network-connectivity/ but then I was not able to apply config to worker nodes. 
Probably due to lack of opened port 50000 from my laptop. When I open the port everything works as expecter

Maybe this should be added to the documentation

**Comment by steverfrancis** [comment]
Good point - access to port 50000 on worker nodes is only needed to apply the initial config, but is definitely needed for that! We'll update the docs, thanks!
